README.md

```diff
@@ -13,6 +13,25 @@ polluting the markup that is delivered to the end user.
 ember install ember-test-selectors
 ```
 
+## Configuartion
+To modify the default configuration, place a block called `ember-test-selectors`
+in your `ember-cli-build.js` file.
+
+### Options
+
+`envsToStrip`
+Defines the environments in which you want the test selectors to be removed.
+By default, selectors are only removed in the `production` environment. You
+might also want to remove them in other staging environments for testing.
+
+```javascript
+var app = new EmberApp({
+  'ember-test-selectors': {
+    envsToStrip: ['production', 'staging']
```

Comment:
I like environments TBH

> Am 05.04.2016 um 00:22 schrieb Ben Limmer <notifications@github.com>:
> 
> In README.md <https://github.com/simplabs/ember-test-selectors/pull/14#discussion_r58459432>:
> 
> > @@ -13,6 +13,25 @@ polluting the markup that is delivered to the end user.
> >  ember install ember-test-selectors
> >  ```
> >  
> > +## Configuartion
> > +To modify the default configuration, place a block called `ember-test-selectors`
> > +in your `ember-cli-build.js` file.
> > +
> > +### Options
> > +
> > +`envsToStrip`
> > +Defines the environments in which you want the test selectors to be removed.
> > +By default, selectors are only removed in the `production` environment. You
> > +might also want to remove them in other staging environments for testing.
> > +
> > +```javascript
> > +var app = new EmberApp({
> > +  'ember-test-selectors': {
> > +    envsToStrip: ['production', 'staging']
> I felt like environments wasn't clear enough. stripInEnvs seems fine to me.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/simplabs/ember-test-selectors/pull/14/files/58d40a78b357ce56ac8613e501c221a1fdd4717f#r58459432>